### ISSUE TITLE
`dotlock activate` does not appear to do anything

### ISSUE THREAD

**Issue by zackmdavis** [alexbecker/dotlock#2]
```
zmd@ReflectiveCoherence:~/Code/Finetooth$ echo $PATH
/home/zmd/.cargo/bin:/home/zmd/bin:/home/zmd/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
zmd@ReflectiveCoherence:~/Code/Finetooth$ dotlock activate
zmd@ReflectiveCoherence:~/Code/Finetooth$ echo $?
0
zmd@ReflectiveCoherence:~/Code/Finetooth$ echo $PATH  # ↓ path is unchanged
/home/zmd/.cargo/bin:/home/zmd/bin:/home/zmd/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
zmd@ReflectiveCoherence:~/Code/Finetooth$ source ./venv/bin/activate  # but activating the virtualenv the old-fashioned way works—
(venv) zmd@ReflectiveCoherence:~/Code/Finetooth$ echo $PATH  # (path changed)
/home/zmd/Code/Finetooth/venv/bin:/home/zmd/.cargo/bin:/home/zmd/bin:/home/zmd/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
```

Of course, dotlock [is just](https://github.com/alexbecker/dotlock/blob/bb4dbfb27ac1ee5db93470d6caecf52ea74f8ade/src/dotlock/activate.py#L5) exec-ing [activate_this.py](https://github.com/pypa/virtualenv/blob/515503c933f4d71df34923e02cd77b2b14744ebf/virtualenv_embedded/activate_this.py)—one imagines that that works _within_ a given Python process, but a user who types `dotlock activate` probably meant to affect their current bash shell, not the emphemeral Python process of the command itself.

**Comment by alexbecker** [comment]
As far as I can tell, the only way to make this work is to spawn a new shell within the `python` process. I don't like this, since I lose my shell history and with e.g. `pipenv` it never quite behaves the same way. So I've replaced `activate` with an actually functional `run` command, which `exec`s programs *within* the `python` process.

**Comment by alexbecker** [comment]
I've released `0.3.0` with `run` in place of `activate`.
